File: specification/networkcloud/resource-manager/Microsoft.NetworkCloud/stable/2023-07-01/examples-js/BareMetalMachines_Uncordon.js
const { NetworkCloud } = require("@azure/arm-networkcloud");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Uncordon the provided bare metal machine's Kubernetes node.
 *
 * @summary Uncordon the provided bare metal machine's Kubernetes node.
 * x-ms-original-file: specification/networkcloud/resource-manager/Microsoft.NetworkCloud/stable/2023-07-01/examples/BareMetalMachines_Uncordon.json
 */
async function uncordonBareMetalMachine() {
  const subscriptionId =
    process.env["NETWORKCLOUD_SUBSCRIPTION_ID"] || "123e4567-e89b-12d3-a456-426655440000";
  const resourceGroupName = process.env["NETWORKCLOUD_RESOURCE_GROUP"] || "resourceGroupName";
  const bareMetalMachineName = "bareMetalMachineName";
  const credential = new DefaultAzureCredential();
  const client = new NetworkCloud(credential, subscriptionId);
  const result = await client.bareMetalMachines.beginUncordonAndWait(
    resourceGroupName,
    bareMetalMachineName
  );
  console.log(result);
}

File: specification/networkcloud/resource-manager/Microsoft.NetworkCloud/stable/2025-02-01/examples-java/BareMetalMachines_Uncordon.java

/**
 * Samples for BareMetalMachines Uncordon.
 */
public final class Main {
    /*
     * x-ms-original-file:
     * specification/networkcloud/resource-manager/Microsoft.NetworkCloud/stable/2025-02-01/examples/
     * BareMetalMachines_Uncordon.json
     */
    /**
     * Sample code: Uncordon bare metal machine.
     * 
     * @param manager Entry point to NetworkCloudManager.
     */
    public static void uncordonBareMetalMachine(com.azure.resourcemanager.networkcloud.NetworkCloudManager manager) {
        manager.bareMetalMachines().uncordon("resourceGroupName", "bareMetalMachineName",
            com.azure.core.util.Context.NONE);
    }
}
